### PR TITLE
WRQ-188: Update internal calls to the updated core/platform APIs

### PR DESCRIPTION
**Pull request by 0x64**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [ ] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
As [core/platform's API is updated](https://github.com/enactjs/enact/pull/3208/), internal codes need to be updated.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated affected API calls.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
No changelog as only internal calls are updated.

### Links
[//]: # (Related issues, references)
WRQ-188
[enactjs/enact PR #3208](https://github.com/enactjs/enact/pull/3208/)

### Comments
Enact-DCO-1.0-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### CODE REVIEW

**Review by juwonjeong — APPROVED** [review]
LGTM

**Review by seunghoh — APPROVED** [review]
LGTM

File: Slider/SliderBehaviorDecorator.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ const SliderBehaviorDecorator = hoc(defaultConfig, (config, Wrapped) => {
 
 		handleDragStart () {
 			// on platforms with a touchscreen, we want to focus slider when dragging begins
-			if (platform.touchscreen) {
+			if (platform.touchScreen) {
```
Comment (seunghoh) [inline Slider/SliderBehaviorDecorator.js:135]: Drag doesn't work on touch mode

Comment (0x64) [reply Slider/SliderBehaviorDecorator.js:135]: Please check the building environment as I successfully build and make it work on several environments.

**Comment by seunghoh** [comment]
LGTM

**Pull request merged**